src/main/java/frc/robot/subsystems/CargoHolder.java

```diff
@@ -34,8 +35,11 @@ public void setIsCargoCollected(boolean isCollected){
   }
 
   public boolean isCargoCollectedStall() {
-    //TODO finish this.
-    return true;
+    //TODO calibrate
+    double actualCurrent = holderMotor.getOutputCurrent();
+    double power = 0.4;
+    double expectedCurrent = 9.375 * power - 0.6384;
+    return actualCurrent - expectedCurrent > DELTA_CURRENT;
```

Comment:
double expectedCurrent = 9.375 * power - 0.6384; 
make this line another private function and call it here. make the numbers to be a final variables.